packages/web3-common/src/errors.ts

```diff
@@ -388,3 +389,7 @@ export class OperationTimeoutError extends Web3Error {
 export class OperationAbortError extends Web3Error {
 	public code = ERR_OPERATION_ABORT;
 }
+
+export class AbiError extends Web3Error {
+	public code = ERR_ABI_ENCODING;
+}
```

Comment:
Why don't these have `constructors`?

Reply:
These use default constructor from the `Web3Error`

Reply:
Every other error in this file is expecting arguments and passing it to the parent constructor as the error string. Are we not expecting additional information to be passed when these errors are created?

Reply:
As of now we don't have any extra information. I am working on next PR with more information in it. Then will add more details to the error.